README.md

```diff
@@ -1,49 +1,30 @@
-# Kind Setup
-From [Kind Ingress Guide](https://kind.sigs.k8s.io/docs/user/ingress/)
-
-```bash
-kind create cluster --name demo-a --config files/kind.yml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-kubectl wait --namespace ingress-nginx \
-  --for=condition=ready pod \
-  --selector=app.kubernetes.io/component=controller \
-  --timeout=90s
-```
+# Armada-Jupyter
 
-```yml
-kind: Cluster
-apiVersion: kind.x-k8s.io/v1alpha4
-nodes:
-  - role: control-plane
-    image: "kindest/node:v1.21.10"
-    kubeadmConfigPatches:
-    - |
-      kind: InitConfiguration
-      nodeRegistration:
-        kubeletExtraArgs:
-          node-labels: "ingress-ready=true"
-    extraPortMappings:
-    - containerPort: 80
-      hostPort: 80
-      protocol: TCP
-    - containerPort: 443
-      hostPort: 443
-      protocol: TCP
-
-  - role: worker
-    image: "kindest/node:v1.21.10"
-```
+Armada Jupyter is a service for deploying JupyterLab K8s Pods onto Armada.
+
+## Prerequisites
+
+Please make sure you have [Armada](https://github.com/G-Research/armada) installed and running.
 
-# Installing
+Ensure that the executor setting has been changed:
 
-```bash
-pip install armada-client
+```yaml
+kubernetes:
+  podDefaults:
+    ingress:
+      hostnameSuffix: "domain.com"
 ```
 
-# Running
+Then make sure that `*.jupyter.domain.com` resolves to your k8s cluster.
+
+## Cluster Preparation
+
+Armada-Jupyter will need a cluster setup with ingress. If you are using kind, you will need to do this differently.
+
+Please follow the [kind](./docs/kind.md) guide to setup your cluster.
+
+## Installation
 
-```bash
-python3 ./src/start.py
+Please following the following guide:
 
-python3 ./src/cancel.py
-```
\ No newline at end of file
+- [Quickstart](./docs/quickstart.md)
```

Comment:
This isn't the guide that you follow?

Reply:
What do you mean?

Reply:
You don't use helm charts to set up your dev environment?

Reply:
Oh - This links to the armada_jupyter quickstart. Not the armada quickstart?

Reply:
haha. yea.